haiku-games/hexvexed/hexvexed-1.0.0.recipe

```diff
@@ -6,11 +6,10 @@ of squares, the tiles are hexagon shaped. It's 1.5x the fun!"
 HOMEPAGE="https://bitbucket.org/scottmc/hexvexed"
 COPYRIGHT="2009-2017 Scott McCreary"
 LICENSE="MIT"
-REVISION="3"
-SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/1.0.0-Beta.zip"
-CHECKSUM_SHA256="d6f9509d6fed6fc0967ec2e095c3874dbe01905395592bfa68bffa2a58341299"
-SOURCE_DIR="scottmc-hexvexed-b8a523a7bf04"
-PATCHES="hexvexed-$portVersion.patchset"
+REVISION="4"
+SOURCE_URI="https://github.com/HaikuArchives/HexVexed/archive/1.0.0-Beta2.tar.gz"
```

Comment:
Beta3 now...

Reply:
bumped recipe to beta3, half way there, images are stored in $dataDir/hexvexed/backgrounds (and are present) but still don't show up, if I copy them to ~/config/non-packaged/... it works

Reply:
https://ibb.co/dPsBBe

Reply:
I guess I'll have to take another look. I have to admit, I didn't try installing HexVexed to /system and if it finds the backgrounds in the data folder there...

Also, shouldn't the settings always be in ~/config/settings/ ?